src/client/api/restful/faucet_enqueue.c

```diff
@@ -58,7 +58,7 @@ int req_tokens_to_addr_from_faucet(iota_client_conf_t const *conf, char const ad
   }
 
   // Check if addr_bech32 has minimum requred length
-  if (strlen(addr_bech32) < BECH32_ED25519_ADDRESS_STR_LEN) {
+  if (strlen(addr_bech32) < BIN_TO_HEX_STR_BYTES(ED25519_PUBKEY_BYTES)) {
```

Comment:
addr_bech32 could be one of ED25519/Alias/NFT addresses.
```suggestion
  if (strlen(addr_bech32) < BIN_TO_HEX_BYTES(ADDRESS_MIN_BYTES)) {
```

Reply:
Fixed.